Gems/ONNX/Code/Include/ONNX/ONNXBus.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Interface/Interface.h>
+
+namespace Ort
+{
+    struct Env;
+    struct AllocatorWithDefaultOptions;
+} // namespace Ort
+
+namespace ONNX
+{
+    struct PrecomputedTimingData
+    {
+        float m_totalPrecomputedRuntime;
+        float m_averagePrecomputedRuntime;
+        int m_totalNumberOfInferences;
+        int64_t m_numberOfCorrectInferences;
+    };
```

Comment:
Can we move this to the Test project? This shouldn't be part of the public bus

Reply:
Changed